### DISCUSSION TITLE
QEN 2B & 7B

### DISCUSSION THREAD

**Discussion by dtubb** [mhumphries2323/Transcription_Pearl#2] [General]
Mark, this looks really cool. I look forward to playing. I and Andrew Janco (@apjanco) have had good success running QWEN 2B and 7B locally to do this sort of work. I have some code here, https://github.com/dtubb/q_transcribe. Andy has been working on using fine-tuning QWEN 7B with some success.

**Comment by mhumphries2323** [comment]
That is great! What kind of accuracy are you getting?

**Reply by dtubb** [reply]
Sorry. I just saw this. Very high, in Spanish and English. Say 95-100%. The trick for QWEN 2B is to give it small chunks of text. Give Andy and I a bit more time, we'll have something to show for it. But, clearly, I don't check GitHub messages enough. If you want, email me at dtubb@unb.ca, and I'll share the code.
